docs/_config.yml

```
@@ -1,12 +1,79 @@
-# Configure generation of web page(s).
-name: OpenSSF Security Baseline
-title: null
+title: Open Source Project Security Baseline
+author:
+  name: OpenSSF Security Baseline SIG
+  email: openssf-sig-security-baseline+subscribe@lists.openssf.org
+logo: /assets/logo.svg
+url: "https://baseline.openssf.org"
+description: > 
+  The Open Source Project Security (OSPS) Baseline is a set of security 
+  criteria that projects should meet to demonstrate a strong security posture. 
```

```suggestion
  controls that projects should meet to demonstrate a strong security posture. 
```

This makes the wording consistent (although we can also fix it later)